src/docs/assets/portal.js

```diff
@@ -4,16 +4,12 @@ import log from "../../lib/log";
 
 const host = (window && window.location && window.location.hostname) || '';
 const parts = host.split('.');
+const GLOBAL_VENDOR_LIST_DOMAIN = 'https://vendorlist.consensu.org/vendorlist.json';
```

Comment:
we could use a distant url from the cmp config object or use this url as default value
someting like 
`const GLOBAL_VENDOR_LIST_DOMAIN = config.vendor_list_source || 'https://vendorlist.consensu.org/vendorlist.json';`
with the import of config

Reply:
👍

Reply:
The config object wouldn't actually be available here because this is going to be running on a separate page from the CMP object. I can however do the global consensu list as a fallback if the original vendors.json path (`./vendors.json`) is not found :)